modules/representation_management/app/swagger/v0/swagger.json

```diff
@@ -189,10 +189,266 @@
             }
           }
         }
+      },
+      "accredited_individual": {
+        "type": "object",
+        "properties": {
+          "data": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "example": "123456"
```

Comment:
this would only be a uuid coming from the accredited_x tables

Reply:
Fixed!